File: web/cse216web/src/domain/entitites/Comment.ts
export class Comment {
    cId: number;
    mId: number; 
    uId: number;
    cContent: String
    constructor(commentId: number, messageId: number, userId: number, commentContent: string) {
        this.cId = commentId;
        this.mId = messageId;
        this.uId = userId;
        this.cContent = commentContent;
    }
}

File: mobile/flutter/my_tutorial_app/lib/models/Comment.dart
class Comment {
  String cContent;
  int cId;
  int mId;
  int uId;
  Comment({
    required this.cContent,
    required this.mId,
    required this.cId,
    required this.uId,
  });

  factory Comment.fromJson(Map<String, dynamic> json) {
    return Comment(
      cContent: json['cComment'],
      cId: json['cId'],
      mId: json['mId'],
      uId: json['uId'],
    );
  }

  Map<String, dynamic> toJson() => {
        'cComment': cContent,
        'cId': cId,
        'mId': mId,
        'uId': uId,
      };
}
